packages/react/src/components/payStream/paystreamPaymentResult/styles.ts

```diff
@@ -0,0 +1,142 @@
+import styled from 'styled-components';
+
+import { rgba } from '../../../utils';
+import { Button } from '../../../ui-kits';
+
+export const StyledPPResultWrapper = styled.div``;
+
+export const StyledPPResultBox = styled.div`
+  background: #fff;
+  border: 1px solid #e3e0e04a;
+  box-shadow: 0 8px 10px rgba(195, 200, 209, 0.2);
+  border-radius: 8px;
+  width: 400px;
+  max-width: 100%;
+`;
+
+export const StyledPPResultContainer = styled.div`
+  padding: 12px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+  color: rgb(128 26 7 / 1);
```

Comment:
those can be passed from theme btw, not but to consider to extend the theme

Reply:
let's do this with class/theme support (next task)